addon/dom-event-listeners.ts

```diff
@@ -98,6 +98,15 @@ export function addEventListener<T extends IDestroyable>(
 ): void {
   assertArguments(target, eventName, callback);
 
+  deprecate(
+    'ember-lifeline addEventListener is deprecated. Please use modifiers instead.',
+    false,
+    {
+      id: 'ember-lifeline-deprecated-addeventlistener',
+      until: '7.0.0',
+    }
```

Comment:
We also need to add `since` and `for` properties (if they aren't present, ember-source@3.27+ will issue a deprecation).

See example commit https://github.com/emberjs/ember-test-helpers/commit/a7d7e3e664572a10d7e44f6ac4974a63df9070cf

Reply:
Added.